ibis/common/collections.py

```diff
@@ -297,6 +299,25 @@ def __reduce__(self) -> tuple:
         return (self.__class__, (dict(self),))
 
 
+@public
+class FrozenOrderedDict(FrozenDict[K, V]):
+    def __init__(self, *args, **kwargs):
+        super(FrozenDict, self).__init__(*args, **kwargs)
+        hashable = tuple(self.items())
```

Comment:
Hash value is different then FrozenDict's hash which uses frozenset() to calculate the hash.